### PR TITLE
Create So.com.xml

### PR DESCRIPTION
**Pull request by gloomy-ghost**

### CODE REVIEW

**Comment by jeremyn** [comment]
Mark as `403`:
- [ ] https://p.ssl.so.com

Mark as `Invalid certificate`:
- [ ] https://maps.so.com

Mark as `Time out`:
- [ ] https://images.so.com
- [ ] https://j.news.so.com
- [ ] https://qcaptcha.so.com
- [ ] https://zhishu.so.com

Add `target`:
- [ ] https://m.image.so.com

Also:
- [ ] Just mark https://t.so.com as `Mixed content`, with no extra comments
- [ ] Change https://api.app.m.so.com from `403` to `Mixed content`. The root URL gets a 403 with both (unencrypted) HTTP and HTTPS. The specific pages I checked from Google have different mixed content blocking problems.
- [ ] For https://tran.news.so.com I get a 403 for both (unencrypted) HTTP and HTTPS. I can't find anything other than the root URL so I don't know if this is the same problem as https://api.app.m.so.com . I think `tran.news.so.com` should be a `target`.

**Comment by gloomy-ghost** [comment]
dev.yunpan.360.cn redirects to https itself, 404 in https though.

**Comment by jeremyn** [comment]
@gloomy-ghost Can you please take a look at this one?

**Comment by gloomy-ghost** [comment]
http://dev.yunpan.360.cn/ (200)
```html
<html>  
<meta http-equiv="refresh" content="0;url=https://dev.yunpan.360.cn/">  
</html> 
```

https://dev.yunpan.360.cn/ (404)

**Comment by jeremyn** [comment]
Are you saying the problem with the `test` should be ignored because the plain HTTP version of the page already does a meta-refresh to the broken HTTPS version of the page?

**Comment by gloomy-ghost** [comment]
There are still some results in search engines, so maybe it just became broken some days ago. I can't tell if it's temporarily or permanently. What would you prefer?

**Comment by jeremyn** [comment]
I'm not sure what you are asking me.

**Comment by gloomy-ghost** [comment]
If it's a temporary error and will be fixed soon, there is no point to comment it and re-enable it days after.

If it's abandoned, I'm not sure do we even need to keep it in comments.

**Comment by J0WI** [comment]
@gloomy-ghost what is the state of this PR?

File: src/chrome/content/rules/360.cn.xml [hunk 1]
```diff
@@ -92,7 +87,11 @@
 	<target host="shouji.360.cn" />
 	<target host="t.360.cn" />
 	<target host="t321.tianji.360.cn" />
+	<target host="wangzhan.360.cn" />
 	<target host="yunpan.360.cn" />
+	<target host="*.yunpan.360.cn" />
+		<test url="http://c2.yunpan.360.cn/" />
+		<test url="http://c12.yunpan.360.cn/" />
```
Comment (J0WI) [inline src/chrome/content/rules/360.cn.xml]: Can you provide one more test url here?

Comment (gloomy-ghost) [reply src/chrome/content/rules/360.cn.xml]: Any numbers from 1 to 13 IIRC. They are shutting down this service shortly, so if there are any issues regarding the yunpan.360.cn subdomain I might just remove them altogether.

Comment (J0WI) [reply src/chrome/content/rules/360.cn.xml]: It's just because we require 3 in our style guide

**Comment by jeremyn** [comment]
It doesn't really matter now, but I've updated the now-completed checklist in https://github.com/EFForg/https-everywhere/pull/7519#issuecomment-260222637 including modifying some of the requested changes.

@J0WI Thanks for taking this one over.

**Pull request merged**